picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -168,13 +168,14 @@
   boolean shutdown;
 
   Picasso(Context context, Dispatcher dispatcher, Cache cache, Listener listener,
-      RequestTransformer requestTransformer, List<RequestHandler> extraRequestHandlers, Stats stats,
-      Bitmap.Config defaultBitmapConfig, boolean indicatorsEnabled, boolean loggingEnabled) {
+      List<RequestTransformer> requestTransformers, List<RequestHandler> extraRequestHandlers,
+      Stats stats, Bitmap.Config defaultBitmapConfig, boolean indicatorsEnabled,
+      boolean loggingEnabled) {
     this.context = context;
     this.dispatcher = dispatcher;
     this.cache = cache;
     this.listener = listener;
-    this.requestTransformer = requestTransformer;
+    this.requestTransformers = Collections.unmodifiableList(new ArrayList<>(requestTransformers));
```

Comment:
refactoring bug?
`this.requestTransformers = requestTransformers != null ? Collections.unmodifiableList(new ArrayList<>(requestTransformers)) : null;`

Reply:
yep, good catch.